client/state/plugins/installed/selectors-ts.ts

```diff
@@ -446,3 +446,25 @@ export const getPluginStatusesByType = createSelector(
 	},
 	( state: AppState ) => state.plugins.installed.status
 );
+
+/**
+ * Returns a Boolean indicating if a site has a particular plugin that
+ * is active. This is useful for jetpack connected sites.
+ *
+ * @param {Object} state - Global state tree
+ * @param {number} siteId - Site ID
+ * @param {string} pluginSlug - Plugin slug
+ * @returns {boolean} - truthiness of a site having an active plugin
+ */
+export const isPluginActive = createSelector(
+	( state: AppState, siteId: number, pluginSlug: string ) => {
+		const sitePlugin = getAllPluginsIndexedBySiteId( state )?.[ siteId ]?.[ pluginSlug ];
```

Comment:
It appears that `getAllPluginsIndexedBySiteId` will always return an object:

https://github.com/Automattic/wp-calypso/blob/aeadcfa0bdb264d4ef8c26a5a72a9b7dcc3e0136/client/state/plugins/installed/selectors-ts.ts#L171

So the first optional chaining looks redundant:

```suggestion
		const sitePlugin = getAllPluginsIndexedBySiteId( state )[ siteId ]?.[ pluginSlug ];
```

Reply:
True!